charts/mibserver/templates/deployment.yaml

```diff
@@ -27,11 +27,17 @@ spec:
       {{- end }}
       serviceAccountName: {{ include "mibserver.serviceAccountName" . }}
       securityContext:
-        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+        fsGroup: 10001
       containers:
         - name: {{ .Chart.Name }}
           securityContext:
-            {{- toYaml .Values.securityContext | nindent 12 }}
+            capabilities:
+              drop:
+                - ALL
+            readOnlyRootFilesystem: {{ ternary "true" "false" ( empty .Values.localMibs.pathToMibs ) }}
```

Comment:
because compile operation on local mibs results in new files creation (index.csv, mib json files, mib asn files) and it requires a read permission to work